packages/react-components/src/components/ParticipantList.tsx

```diff
@@ -77,6 +77,8 @@ export type ParticipantListProps = {
   onRemoveParticipant?: (userId: string) => void;
   /** Optional callback to render custom menu items for each participant. */
   onFetchParticipantMenuItems?: ParticipantMenuItemsCallback;
+  /** Optional callback when rendered ParticipantItem is clicked */
```

Comment:
Is it true that if this is defined then `onFetchParticipantMenuItems` is ignored? If so, please note this in the docstring.

Also, what is our plan for supporting custom participant menu props set by Contoso in the composite API? They have to passed in to the drawer now, i.e., we have to convert from IContextMenuItem -> _DrawerMenuItem in the composite layer. This feels clunky -- on desktop `ParticipantList` handles the per-particient menu, but on mobile it doesn't.

I don't have a solution here, simply complaints :(

Reply:
Sorry for the confusion here. `onFetchParticipantMenuItems` is not affected regardless of `onParticipantClick` being defined or not. `onParticipantClick` is meant to add additional behavior Contoso may want to add. 

If Contoso wanted to support custom participant menu props for participants in the `ParticipantList`, they would use the prop `onFetchParticipantMenuItems`. But if they wanted to use the _DrawerMenu to duplicate the custom participant menu props I think they would have to do the conversion of each IContextMenuItem to _DrawerMenuItem. Perhaps we can unify them in some way. Do we plan on exposing DrawerMenu in our API the future? @JamesBurnside If not we don't have to worry.

Reply:
This is a good example of where we designed our API to be too open and hence inflexible -- lets make a task to support `onFetchParticipantMenuItems` on mobile and bring this up in feature weekly as the solution could be messy

Reply:
This would cause further divide in call -> callWithChat but perhaps we limit the `onFetchParticipantMenuItems` interface for call with chat composite to be the intersection of IContextMenuItem and _DrawerMenuItem interfaces (_DrawerMenuItem is based of IContextMenuItem so they are similar)

Reply:
I think we can avoid this by converting ContextualMenuItem to _DrawerMenuItem ourselves if mobile view. We access the menuItems of `ParticipantItem` through a callback. The change to this branch would be this commit: https://github.com/Azure/communication-ui-library/commit/335dd81e9123207afe24c809bf03a66047d3070b

Reply:
Can we land this PR first @prprabhu-ms? Currently `onFetchParticipantMenuItems ` prop is not used in CallWithChat composite for the `ParticipantList`. I will add its usage to create `IContextualMenuItem` and `_DrawerMenuItem` in another PR for better readability.